pages/index.js

```diff
@@ -173,7 +173,7 @@ export default function Home() {
           {/* <!-- Contact Form  --> */}
           <div data-aos="zoom-in" id='contacts'>
             <h2 className={styles.grabh2}>Get In Touch With Us</h2>
-            <form onSubmit={onSubmit}>
+            <form action="https://formsubmit.co/{Enter Email}" method="POST">
```

Comment:
how we receive the mail?

Reply:
@AmanJain18 asked me to put it this way

And you would not get email currently, you have to put your correct email there.
I have explained this part in Additional Notes of this PR above.

**I asked for the official email (to put there) but I wasn't provided.**

Reply:
ok